toffy/normalize.py

```diff
@@ -521,6 +521,7 @@ def normalize_fov(img_data, norm_vals, norm_dir, fov, channels, extreme_vals):
 
     # correct images and save
     normalized_images = img_data / norm_vals.reshape((1, 1, 1, len(norm_vals)))
+    normalized_images = normalized_images.astype(np.float32)
```

Comment:
If you cast `norm_vals` to be the same dtype as `img_data` before dividing, does that accomplish the same thing? Then nothing needs to be hardcoded

Reply:
yep just had to adjust the test data to not use the int8 default